samples/sample_common/src/sample_utils.cpp

```diff
@@ -152,7 +174,7 @@ void CSmplYUVReader::Reset()
     }
 }
 
-mfxStatus CSmplYUVReader::LoadNextFrame(mfxFrameSurface1* pSurface)
+mfxStatus CSmplYUVReader::LoadNextFrame(mfxFrameSurface1* pSurface, mfxU32 displayOrder)
```

Comment:
now you are breaking backward compatibility for all samples. Are you sure that every sample is prepared to deal with this function taking 2 parameters instead of 1?

Reply:
created another function for change position in file